Makefile

```diff
@@ -197,6 +197,7 @@ get-deps: get-deps-tests get-deps-x-tests get-deps-codegen get-deps-verify
 get-deps-tests:
 	@echo "go get SDK testing dependencies"
 	go get github.com/stretchr/testify
+    go get github.com/google/go-cmp/cmp
```

Comment:
Need to make sure to use tabs here instead of space. Makefiles require tabs for tasks.